ui/packages/catalog-ui-search/src/main/webapp/js/model/QueryResponse.js

```diff
@@ -47,6 +47,24 @@ function humanizeResourceSize(result) {
     }
 }
 
+function handleResultFormFields(result, selectedResultTemplate) {
+    if (selectedResultTemplate) {
+        let resultAttributes = selectedResultTemplate.descriptors;
+        if (resultAttributes.length > 0) {
+            let newProperties = {};
+            newProperties.id = result.metacard.properties.id;
+            newProperties.title = result.metacard.properties.title ? result.metacard.properties.title : '';
+            newProperties["metacard-type"] = result.metacard.properties["metacard-type"];
```

Comment:
Do we need tags as well?

Reply:
They somehow were already being included in the query response, but i suppose it wouldn't hurt to explicitly add them here

Reply:
done